<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbKey">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCategories">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Categories</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbOU">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OU</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbTuition">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuition</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbLivingExpense">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Living Expense</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbOthers">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Others</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtQuestion">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="listCheckBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>289</width>
       <height>119</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGUI_example">
    <property name="title">
     <string>GUI example</string>
    </property>
   </widget>
   <addaction name="menuGUI_example"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
